### PR TITLE
Express api to receive contact form data

### PR DESCRIPTION
**Pull request by TilteD24**
### Related Issue
- Issue number goes here: #184 

### Proposed Changes 
- Added app.js
- Added Express dependency

### Checklist 
- [ ] Documentation of the code 
- [ ] Add your name to the CONTRIBUTORS file 
- [x] Testing your code in the local machine

### CODE REVIEW

**Review by jay-io — CHANGES_REQUESTED** [review]
Thank you for contributing! Also, please follow the discussion thread in Discord.
- You were supposed to change the files inside `/discord` directory, not in the root.
- You were not supposed to overwrite the default form submit action. The mail has to stay, and there is to be an additional hyperlink that triggers the call to the API (refer to the messages pinging you on Discord)

I can also see you have changed the other files at some places refactoring the code. While we appreciate this, we do not recommend it, since your commit messages don't relate to the changes you're making leading to poor documentation. So, avoid them.

File: app.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const router = express.Router();
+const bodyParser = require("body-parser");
+const app = express();
+app.use("/", router);
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
```
Comment (jay-io) [inline app.js:8]: > In Express >= 4.16, body parser has been re-added under the methods express.json(). The replacement for the above question is
> 
> ```js
> const express = require("express");
> const app = express();
> app.use(express.json());
> app.use(express.urlencoded({ extended: true }));```

You can omit installing the `body-parser` package now. Refer to [this](https://stackoverflow.com/questions/55558402/what-is-the-meaning-of-bodyparser-urlencoded-extended-true-and-bodypar) thread for more information.

**Comment by TilteD24** [comment]
> I can also see you have changed the other files at some places refactoring the code. While we appreciate this, we do not recommend it, since your commit messages don't relate to the changes you're making leading to poor documentation. So, avoid them.

I didn't refactor files. I don't know how they show up in the commit.

**Comment by jay-io** [comment]
@TilteD24 You probably have [code format on save](https://blog.yogeshchavan.dev/automatically-format-code-on-file-save-in-visual-studio-code-using-prettier) enabled if you're using VSCode and extensions like Prettier. Try disabling them at User level.

**Comment by jay-io** [comment]
@TilteD24 I think it'll be safe and more convenient for you to close this PR and create a new one with the required changes. Take a backup of your branch, delete it & create a new one and overwrite it with only the app.js file content in the place and the file it's supposed to be in.

**Pull request closed without merging**